plugins/modules/digital_ocean_block_storage.py

```diff
@@ -286,9 +326,9 @@ def main():
     try:
         handle_request(module)
     except DOBlockStorageException as e:
-        module.fail_json(msg=e.message, exception=traceback.format_exc())
```

Comment:
`Exception.message` doesn't exist on python3. But `str(Exception)` achieves the same, in all Python versions

See similar discussion in other projects for context: https://github.com/google/yapf/issues/564